app/views/partials/view-claim/overview-summary.html

```diff
@@ -0,0 +1,37 @@
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set visitRules %}
+  <span class="govuk-caption-xl">Visit rules</span>
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-4">{{ displayHelper.getRegionRulesByValue(Claim['Country']) }}</h1>
```

Comment:
Heading needs to be meaningful, so caption text would go inside the header tag and restyle there. Think about what a screenreader would say out loud 🙂